javascript/training-installation.rst

```diff
@@ -2,13 +2,13 @@
 Setup
 =====
 
-To get Plone and example packages for this training installed, please follow the installation instructions about `how to get setup <https://training.plone.org/5/plone_training_config/instructions.html>`_
-
-After that, issue the following command to get the development environment for the ``mockup-minimalpattern`` example package installed
+To get Plone and example packages for this training installed, please clone `collective.jstraining <https://github.com/collective/collective.jstraining>`_ and install Plone by executing the following commands:
```

Comment:
Rewrite sentence for brevity:

```
To install Plone and example packages for this training, first clone `collective.jstraining <https://github.com/collective/collective.jstraining>`_, then execute the following commands:
```

Reply:
Totally missed this page at first and this line was actually incorrect, thanks for pointing that out

Reply:
This one got missed.